integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -9,238 +9,215 @@
+  serviceName: {{ $masterName | quote }}
+  replicas: {{ $root.Values.master.replicas }}
   template:
     metadata:
       labels:
-        name: {{ template "alluxio.fullname" . }}-master
-        app: {{ template "alluxio.name" . }}
-        chart: {{ template "alluxio.chart" . }}
-        release: {{ .Release.Name }}
-        heritage: {{ .Release.Service }}
+        name: {{ $fullName }}-{{ $masterName }}
+        app: {{ $name }}
+        chart: {{ $chart }}
+        release: {{ $release.Name }}
+        heritage: {{ $release.Service }}
         role: alluxio-master
     spec:
-      hostNetwork: true
-      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: {{ $root.Values.master.hostNetwork }}
+      dnsPolicy: {{ $root.Values.master.dnsPolicy }}
       nodeSelector:
-      {{- if .Values.master.nodeSelector }}
-{{ toYaml .Values.master.nodeSelector | trim | indent 8  }}
-      {{- else if .Values.nodeSelector }}
-{{ toYaml .Values.nodeSelector | trim | indent 8  }}
+      {{- if $root.Values.master.nodeSelector }}
+{{ toYaml $root.Values.master.nodeSelector | trim | indent 8  }}
+      {{- else if $root.Values.nodeSelector }}
+{{ toYaml $root.Values.nodeSelector | trim | indent 8  }}
       {{- end }}
-      selector:
-        matchLabels:
-          app: {{ template "alluxio.name" . }}
-          role: alluxio-master
-          name: {{ template "alluxio.fullname" . }}-master
       securityContext:
-        runAsUser: {{ .Values.user }}
-        runAsGroup: {{ .Values.group }}
-        fsGroup: {{ .Values.fsGroup }}
+        fsGroup: {{ $root.Values.fsGroup }}
+      {{- if and $needJournalVolume (not $isRoot) }}
       initContainers:
-        - name: format-master-if-need
-          image: {{ .Values.image }}:{{ .Values.imageTag }}
-          imagePullPolicy: {{ .Values.imagePullPolicy }}
-          resources:
-            limits:
-            {{- if .Values.master.resources  }}
-              {{- if .Values.master.resources.limits }}
-                {{- if .Values.master.resources.limits.cpu  }}
-              cpu: {{ .Values.master.resources.limits.cpu }}
-              memory: {{ .Values.master.resources.limits.memory }}
-                {{- end }}
-              {{- end }}
-            requests:
-              {{- if .Values.master.resources.requests }}
-                {{- if .Values.master.resources.requests.cpu  }}
-              cpu: {{ .Values.master.resources.requests.cpu }}
-              memory: {{ .Values.master.resources.requests.memory }}
-                {{- end }}
-              {{- end }}
-            {{- end }}
-          envFrom:
-            - configMapRef:
-                name: {{ template "alluxio.name" . }}-config
-          env:
-            - name: ALLUXIO_WORKER_HOSTNAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: status.hostIP
-          volumeMounts:
-            - name: tz-config
-              mountPath: /etc/localtime
```

Comment:
@cheyang  I chose to remove this localtime because these templates aim at giving users a good place to start with Alluxio. So we design the templates to serve the most basic use cases. `/etc/localtime` seems very specific to some use cases me so I chose to remove it for the simplicity.

Mounting `/etc/localtime` is definitely a good trick. Potentially if many users are hitting the time sync issue (for example, in logs), we should add this to the documentation or Alluxio K8s specific FAQ page.